Makefile

```diff
@@ -33,6 +33,20 @@ test:: install_plugins
 	PULUMI_TERRAFORM_BRIDGE_TEST_PROVIDER=$(shell pwd)/bin/pulumi-terraform-bridge-test-provider \
 		go test -count=1 -coverprofile="coverage.txt" -coverpkg=./... -timeout 2h -parallel ${TESTPARALLELISM} ./...
 
+test_shard:: install_plugins
```

Comment:
This also needs a comment explaining what it does, and that it is provided for CI.